java/src/main/resources/application.properties

```diff
@@ -0,0 +1,28 @@
+server.port=8080
+spring.profiles.active=develop
+
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=false
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.properties.hibernate.format_sql=false
+
+
+spring.datasource.type=com.alibaba.druid.pool.DruidDataSource
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://aurora-mysql-ccc.cluster-cgz4ugkbn4yc.ap-northeast-2.rds.amazonaws.com:3306/ccc?useUnicode=true&characterEncoding=utf8
+spring.datasource.username=admin
+spring.datasource.password=Jinfei!21
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- ad2a4fd9-030a-41d6-af3a-c601d91c252d -->

Reply:
please help me to delete it,thanks, i have deleted the git project

Reply:
delete this pull request, thanks very much